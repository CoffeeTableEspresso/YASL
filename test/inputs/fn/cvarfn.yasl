echo collections.set()
echo collections.set(1)
echo collections.set(1, 2)
echo collections.set(1, 2, 3)
echo collections.set(1, 2, 3, 4)
echo collections.set(1, 2, 3, 4, 5)
echo collections.set(1, 2, 3, 4, 5, 6)
echo collections.set(1, 2, 3, 4, 5, 6, 7)
echo collections.set(1, 2, 3, 4, 5, 6, 7, 8)
echo collections.set(1, 2, 3, 4, 5, 6, 7, 8, 9)
