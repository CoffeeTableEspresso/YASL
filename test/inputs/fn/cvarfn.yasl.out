set()
set(1)
set(2, 1)
set(2, 1, 3)
set(2, 1, 4, 3)
set(2, 1, 4, 3, 5)
set(2, 1, 4, 3, 6, 5)
set(2, 1, 4, 3, 7, 6, 5)
set(2, 1, 4, 3, 7, 8, 6, 5)
set(2, 1, 4, 3, 7, 8, 6, 9, 5)
